test/tests/steel_creep_damage_oh/power_law_creep.i

```diff
@@ -0,0 +1,190 @@
+# This model is activated with cli_args to compare the response with a single model with that of two models
```

Comment:
Well, I meant to put this right next to the creep_twenty model in the input file, but it's fine.